src/Middleware/Localization/src/RequestLocalizationMiddleware.cs

```diff
@@ -169,22 +169,29 @@ private static void SetCurrentThreadCulture(RequestCulture requestCulture)
 
     private static CultureInfo? GetCultureInfo(
         StringSegment cultureName,
-        IList<CultureInfo> supportedCultures,
+        IList<CultureInfo>? supportedCultures,
         bool fallbackToParentCultures,
         int currentDepth)
     {
+        // If the cultureName is an empty string there
+        // is not chance we can resolve the culture info.
+        if (cultureName == "")
```

Comment:
Why?!! this is equivalent to `InvariantCulture`

Reply:
```CultureInfo.GetCultureInfo(string.Empty) == CultureInfo.InvariantCulture``` returns false.

Reply:
If we don't drop out when the string is empty we'll uselessly spin down the max depth. If it is an empty string we can just return null.

Reply:
Use `Equals` instead